$MirOS: ports/mailnews/pine/patches/patch-imap_src_imapd_imapd_8,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
$OpenBSD: patch-imapd.8c,v 1.4 2003/10/05 22:00:00 jakob Exp $
--- imap/src/imapd/imapd.8.orig	Tue May 18 17:46:32 2004
+++ imap/src/imapd/imapd.8	Wed May  4 23:28:07 2005
@@ -3,7 +3,7 @@
 .SH NAME
 IMAPd \- Internet Message Access Protocol server
 .SH SYNOPSIS
-.B /usr/etc/imapd
+.B imapd
 .SH DESCRIPTION
 .I imapd
 is a server which supports the
